$MirOS: ports/net/rtorrent/patches/patch-doc_rtorrent_1,v 1.2 2006/03/18 18:26:37 tg Exp $
--- doc/rtorrent.1.orig	Fri Sep 29 11:56:12 2006
+++ doc/rtorrent.1	Fri Oct  6 16:39:08 2006
@@ -3,7 +3,7 @@
 .\" <http://shell.ipoline.com/~elmert/comp/docbook2X/> 
 .\" Please send any bug reports, improvements, comments, patches, 
 .\" etc. to Steve Cheng <steve@ggi-project.org>.
-.TH "RTORRENT" "1" "15 August 2006" "BitTorrent client for ncurses" ""
+.TH "RTORRENT" "1" "15 August 2006" "net/rtorrent" ""
 
 .SH NAME
 rtorrent \- a BitTorrent client for ncurses
@@ -140,7 +140,7 @@ Display help and exit.
 Set the address reported to the tracker.
 .TP
 \fB-n\fR
-Don't load ~/.rtorrent.rc on startup.
+Don't load ~/.etc/rtorrent.rc on startup.
 .TP
 \fB-o key1=opt1,...\fR
 .TP
